doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,401 @@
+# Dataverse 6.2
+
+Please note: To read these instructions in full, please go to https://github.com/IQSS/dataverse/releases/tag/v6.2 rather than the list of releases, which will cut them off.
+
+This release brings new features, enhancements, and bug fixes to the Dataverse software.
+Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## 💡Release Highlights
+
+### Search and Facet by License  
+
+A new search facet called "License" has been added and will be displayed as long as there is more than one license in datasets and datafiles in browse/search results. This facet allow you to filter by license such as CC0, etc.
+
+Also, the Search API now handles license filtering using the `fq` parameter, for example : `/api/search?q=*&fq=license%3A%22CC0+1.0%22` for CC0 1.0. 
+
+For more information, see [#10204](https://github.com/IQSS/dataverse/issues/10204).
+
+### Return to Author Now Requires a Reason
+
+The Popup for returning to author now allows to type in a message to explain the reasons of return and potential edits needed, that will be sent by email to the author.
+
+Please note that this note is mandatory, but that you can still type a creative and meaningful comment such as "The author would like to modify his dataset", "Files are missing", "Nothing to report" or "A curation report with comments and suggestions/instructions will follow in another email" that suits your situation.
+
+<a id="multiple-pid-sup"></a>
+### Support for Using Multiple PID Providers
+
+This release adds support for using multiple PID (DOI, Handle, PermaLink) providers, multiple PID provider accounts
+(managing a given protocol, authority,separator, shoulder combination), assigning PID provider accounts to specific collections,
+and supporting transferred PIDs (where a PID is managed by an account when its authority, separator, and/or shoulder don't match
+the combination where the account can mint new PIDs). It also adds the ability for additional provider services beyond the existing
+DataCite, EZId, Handle, and PermaLink providers to be dynamically added as separate jar files.
+
+These changes require per-provider settings rather than the global PID settings previously supported. While backward compatibility 
+for installations using a single PID Provider account is provided, updating to use the new microprofile settings is highly recommended
+and will be required in a future version.
+
+[New microprofile settings](#microprofile-settings)
+
```

Comment:
Can we link to the updated guides section on configuring PIDs?

Reply:
There is one setting from there that doesn't belong to the same PR but let me look at this.

Reply:
Is this the right link? https://dataverse-guide--10432.org.readthedocs.build/en/10432/installation/config.html#global-settings